modules/installation-aws-upi-bootstrap.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-bootstrap-{context}"]
+= Initializing the bootstrap node on AWS UPI
+
+After you create all of the required infrastructure in Amazon Web Services (AWS),
+you can install the cluster.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and worker roles.
+* Create the bootstrap node.
+* Create the control plane nodes.
+* If you plan to manually manage the worker machines, create the worker machines.
+
+.Procedure
+
+. Change to the directory that contains the installation program and run the
+following command:
++
+----
+$ ./openshift-install user-provided-infrastructure bootstrap-complete --dir=<installation-directory> <1>
+----
+<1> Specify the directory name that you generated the `install-config.yaml` file
+and ignition config files for your cluster.
++
+After a message displays that contains `bootstrap-complete`, your bootstrap node
+has initialized.
```

Comment:
Folks should be considering exit codes, not logged messages.  We may change logged messages later, but "exited zero" is always going to be a stable success marker.  If you want a more-stable log marker, you might say "If the command exits without a FATAL warning, your production control-plane has initialized" (or something, I'm leaning on the openshift/installer#1568 wording).